package.json

```diff
@@ -35,7 +35,7 @@
     "socket.io-client": "^1.4.5"
   },
   "devDependencies": {
-    "aframe": "0.5.0",
+    "aframe": "0.7.0",
```

Comment:
Did this to verify that the tests pass with 0.7.0. Not sure if you intentionally kept it at 0.5.0,

Reply:
This was just outdated, it wasn't intentional